### ISSUE TITLE
DateTimePicker wrong time direction

### ISSUE THREAD

**Issue by viict** [dmtrKovalenko/material-ui-pickers#775]
## Environment

| Tech                | Version |
|---------------------|---------|
| material-ui-pickers |    2.0.2     |
| material-ui         |     3.5.1    |
| React               |    16.6.3     |
| Browser             |    chrome  70     |
| Peer library        |    moment     |

## Steps to reproduce
1. Create a DateTimePicker with 24 hours display
2. Open it

## Expected behavior
The time show in order, hours and minutes;

## Actual behavior
The time is shown wrongly, showing minutes first and hours later

## Example

I found that if I change direction to "rtl" it works as expected, but, everything else breaks.

![image](https://user-images.githubusercontent.com/529813/48839999-7d5c7280-ed74-11e8-9f11-446ac68fbba5.png)

**Comment by dmtrKovalenko** [comment]
That sucks!!!

**Comment by viict** [comment]
@dmtrKovalenko i'm using that fix on my app, but, as reported, only checked using moment

**Comment by dmtrKovalenko** [comment]
Seems like now direction is wrong for users that don’t use jss provider

**Comment by viict** [comment]
The #780 appeared to be using Provider, but, as I have more time now, if you want I can take a look at that doing more tests as I will change the version on my app anyway.

**Comment by Thomas-X** [comment]
For the people looking for a datetime picker, picking one finally and encountering this bug: you can use the following bit to override the CSS that causes the reversing using mui's themes.

```js
const theme = createMuiTheme({
	MuiPickerDTHeader: {
		hourMinuteLabel: {
			flexDirection: 'row',
		}
	}
	// ... dont forget your palette definitions here as well, see:
	// https://material-ui.com/style/color/
	// https://material-ui.com/customization/themes/
});
```

**Comment by viict** [comment]
@dmtrKovalenko I don't know why, but, after the merge request on https://github.com/dmtrKovalenko/material-ui-pickers/commit/9f77ae9ea1c6cb2bc9c0e5c38181d218e8b97a8c 
https://github.com/dmtrKovalenko/material-ui-pickers/blob/9f77ae9ea1c6cb2bc9c0e5c38181d218e8b97a8c/lib/src/DateTimePicker/components/DateTimePickerHeader.tsx#L146

the next commit https://github.com/dmtrKovalenko/material-ui-pickers/commit/141539c0710dd053c3e11dd937edd03ec7b4145d appears to revert the file (and apparently the history). 

https://github.com/dmtrKovalenko/material-ui-pickers/blob/141539c0710dd053c3e11dd937edd03ec7b4145d/lib/src/DateTimePicker/components/DateTimePickerHeader.tsx#L146

Can you check on your side?

**Comment by dmtrKovalenko** [comment]
@viict I will check. We should definitely fix that asap

**Comment by dmtrKovalenko** [comment]
Completely fixed in 2.0.4
